src/components/NotFoundController.jsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * NotFoundController simply renders the NotFound component. This controller is
+ * necessary for transitions. Without it, the component will not recieve the
+ * function visibleTransitionOut and the page will transition will fail.
+ */
+
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+
+// Components
+import NotFound from 'components/NotFound';
+
+export default class NotFoundController extends FalcorController {
+  static getFalcorPathSets() {
```

Comment:
Oh I see, this is just a thin wrapper. But can you just directly render the Notfound component and get rid of this controller?